ui/selectors/assets.ts

```diff
@@ -157,7 +162,7 @@ export const getMultiChainAssets = createDeepEqualSelector(
   getAccountAssets,
   getAssetsMetadata,
   getAssetsRates,
-  (_state) => _state.metamask.conversionRates,
+  getMultichainConversionRateSelector,
```

Comment:
Why are we using these rates ? It's shouldn't be relevant for the new multichain logic :thinking: 

https://github.com/MetaMask/metamask-extension/blob/main/ui/selectors/multichain.ts#L462-L471

Reply:
I don't understand why we have 2 source for therse, the `MultichainAssetRatesController` should provide everything needed.

Reply:
The main coin-overview component is using rates from the rates controller; https://github.com/MetaMask/metamask-extension/blob/bab4d501c2dfb2f0070aadca819439578771bae3/ui/hooks/useCurrencyDisplay.js#L153

While the assetList, initially was using rates from the multichainAssetsRatesController. 
The values displayed for the native token were slightly off, because the ratesController uses cryptocompare while the multichainAssetsRatesController uses price-api; 
The motivation behind this change was to align the values we display in both components;

I agree with the point to have a single controller, in this case, multichainAssetsRatesController to display the native rates values, but we would also need to touch some of the EVM logic which we did not want to include as part of this PR.

Reply:
Then we should update the coin overview and not add a patch here, we're adding a unwanted logic to a brand new selector this seems wrong to me.

Reply:
We also wanted to avoid updating that logic to not break the BTC logic, can look more 👀

Reply:
Let me align with the BTC team on this

Reply:
After further discussion, we can break the BTC support as long as the extension doesn't hard crash (it shouldn't). The BTC snap will start working as expected again when it updates the the latest API

Reply:
And for the coin overview we shouldn't be basing on it for this implementation. This PR should only use the new API and we should follow-up with a PR to fix the coin-overview so it uses the new `MultichainAssetsRateController`.